src/twisted/topfiles/9128.feature

```diff
@@ -0,0 +1 @@
+twisted.internet._sslverify now enables TLS 1.3 cipher suites.
```

Comment:
So, sorry about this, but generally we try not to reference the complete path of private modules. It'd be better to say something like:

`twisted.protocols.tls.TLSMemoryBIOProtocol and anything that uses it indirectly including the TLS client and server endpoints now enables TLS 1.3 cipher suites.`